.github/workflows/release.yml

```diff
@@ -206,9 +219,11 @@ jobs:
         run: bash tools/python_build.sh -i
 
       - name: Publish Python package
+        env:
+          DRY_RUN_FLAG: ${{ inputs.dry_run && '-n' || '-nn' }}
```

Comment:
What does `-n` and `-nn` have to do with dry release? The `n`'s seems like a random choice.